OxfordBodleian/Juel-Jensen/BDLaethe30.xml

```diff
@@ -0,0 +1,865 @@
+                                    <hi rend="rubric">ውዳሴ፡ ወግናይ፡ እሙ፡ <seg rend="above">ለአዶናይ፡</seg> ቅድስት፡ ወብፅዕት፡</hi> ስብሕት፡ ወቡርክት፡ ክብ<cb n="b"/>ርት፡ ወልዕልት፡ አንቀጸ፡ ብርሃን፡ 
+                                    ማዓርገ፡ ሕይወት፡ ወማኅደረ፡ መለኮት፡ ቅድስተ፡ ቅዱሳን። አንቲ፡ ውእቱ፡ ኦእግዝእትነ፡ ወላዲተ፡ አምላክ፡ ማርያም፡ ድንግል፡ ተሰመይኪ፡ 
+                                    ሥምረተ፡ አብ፡
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    <locus target="#136vb"/>
+                                    አንቲ፡ በአማን፡ ዘኮንኪ፡ ምክህ፡ ለዘምደ፡ ክርስቲያን፡ ወብኪ፡ ቅሩባነ፡ ኮነ፡ እምድር፡ ለማኅደር፡ ውስተ፡ አርያም፡ በእንተ፡ ተሠግዎቱ፡ 
+                                    ለወልደ፡ አምላክ፡ አምኔኪ፡ ስዕለት፡ ለነፍሳቲነ፡ ሕይወተ፡ ኦጸጋዊተ፡ ምሕረት፡ ለእለ፡ የ
+                                </explicit>
+                                <note>The text ends abruptly due to material loss.</note>
+                            </msItem>
+                        </msItem>
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">iii + 137</measure>
+                                    <measure unit="leaf" type="blank">1</measure>
+                                    <locus target="#iiiv"/>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>174</height>
+                                        <width>113</width>
+                                        <depth>63</depth>
+                                    </dimensions>
+                                    <dimensions type="folio" unit="mm">
+                                        <height>173</height>
+                                        <width>110</width>
+                                    </dimensions>
+                                    <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#33r"/>.</note>
+                                    <measure type="weight" unit="g">804</measure><!--with cloth cover-->
+                                </extent>
+                                <foliation>Pencil foliation in the upper right corner carried out by the Bodleian Library during our stay in January 2020. 
+                                    </foliation>
+                                 <collation>
+                                     <signatures>
+                                         <note>
+                                             Quire marks are written in the upper inner margin of the first leaf of each quire. Some of them are barely legible or nearly illegible.                                                                               
```

Comment:
I'd pick one or the other, they have the same meaning "barely legible or nearly illegible" :)